pkg/network/master/egressip.go

```diff
@@ -240,3 +243,90 @@ func (eim *egressIPManager) SetNamespaceEgressDropped(vnid uint32) {
 
 func (eim *egressIPManager) SetNamespaceEgressViaEgressIPs(vnid uint32, activeEgressIPs []common.EgressIPAssignment) {
 }
+
+type egressNetworkPolicyCache struct {
+	*sync.Mutex
+	// namespace name -> egress network policy name -> egressNetworkPolicy
```

Comment:
FTR, it's illegal to have more than one EgressNetworkPolicy in a namespace, so you could remove a level here